update: decky-game-settings: 1.2.x

Josh5 opened the pull request:
<!--
    READ ALL COMMENTS IN THIS TEMPLATE BEFORE SUBMITTING YOUR PR!

    This template is for updating an existing plugin on the store WITHOUT changing the repository it is linked to. If you are taking any other action, please start over by creating a new pull request and selecting the appropriate template.

    If you have not already, please read the review and testing page on the wiki.
    https://wiki.deckbrew.xyz/plugin-dev/review-and-testing

    Before submitting, make sure you have done the following:
    - Tested two other plugins and left feedback on their PRs as described below.
    - Replaced REPLACE_WITH_PLUGIN_NAME and REPLACE_WITH_SUMMARY with the appropriate information.
    - Filled out the Task Checklist, including the Yes/No questions.
    - Deleted the unnecessary testing checkbox.
-->

# Deck Settings

<!--
    Include a detailed summary of what updates you are adding to the plugin, attaching images or videos if necessary. If this is an urgent update (ex. security patch), please explain the urgency and notify the Loader Team on Discord.
-->

UPDATES:

- Added in-app report submission with device/system detection and screenshot uploading.
- UI improvements


## Task Checklist

<!--
    For checkboxes, change [ ] to [x] to check the box.
    For Yes/No questions, replace "Yes/No" with "Yes" or "No".
-->

### Developer

- [x] I am the original author or an authorized maintainer of this plugin.
- [x] I have abided by the licenses of the libraries I am utilizing, including attaching license notices where appropriate.

### Plugin

- [x] I have verified that my plugin works properly on the Stable and Beta update channels of SteamOS.
- [x] I have verified my plugin is unique or provides more/alternative functionality to a plugin already on the store.

### Backend

- **No**: I am using a custom backend other than Python.
- **No**: I am using a tool or software from a 3rd party FOSS project that does not have it's dependencies [statically linked](https://en.wikipedia.org/wiki/Static_library).
- **No**: I am using a custom binary that has all of it's dependencies statically linked.

### Community

<!--
    Please submit both of your testing reports before creating your PR. You will need to link the comments you left in a comment on this PR.

    If no plugin additions or updates are ready for testing at this time, then you may ignore this checkbox. You may be asked to test new plugins as they are submitted.

    Your PR will be prioritized lower if you do not test other plugins, but this step is optional.
-->

- [ ] I have tested and left feedback on two other [pull requests][pulls] for new or updating plugins.
- [ ] I have commented links to my testing report in this PR.

## Testing

<!--
    DO NOT FORGET THIS STEP! Otherwise, testers may incorrectly test your plugin.

    If your plugin uses the provided Python backend and React frontend, your plugin must be tested on the Stable or Beta update channel of SteamOS. REMOVE the line with the Preview checkbox below.

    If your plugin uses a custom backend or pre-build binaries without statically linked dependency (ex. glibc), your plugin must be tested on the SteamOS Preview update channel. REMOVE the line with the Stable or Beta checkbox below.

    DO NOT CHECK THIS BOX YOURSELF UNLESS SOMEONE HAS COMMENTED A TESTING REPORT FOR YOUR PLUGIN! This will lead to delays in the review process.
-->

- [ ] Tested by a third party on SteamOS Stable or Beta update channel.

[pulls]: https://github.com/steamdeckHomebrew/decky-plugin-database/pulls?q=is%3Apr+is%3Aopen+sort%3Acreated-desc+-status%3Afailure+-draft%3Atrue+-author%3A%40me

beebls reviewed (commented):
Update all looks good, just want to get a second opinion on if leaving the debug logs and boilerplate python methods is alright

Josh5 commented:
@beebls I have removed or commented out a bunch of the console logging and rebased the PR branch to the head of `main`.
Does that look better to you?

MegalonVII commented:
# Plugin Testing Report

## System Specs
- SteamOS 3.7.15 (Stable)
- Steam version 1759461205
- Decky version 3.1.11 (Stable)

## Installed Plugins
- Deck Settings - 1.2.5-976d6fb
- CSS Loader - 2.1.2
- Audio Loader - 1.6.1
- Controller Tools - 2.0.2
- MagicPods - 2.0.8
- EmuDecky 1.0.8
- Game Theme Music - 1.7.1-96106b0
- Decky Framegen - 0.11.15
- Decky Lossless - 0.11.0
- Wine Cellar - 0.1.6
- AutoFlatpaks - 1.6.8
- TabMaster - 2.11.0
- MagicBlack - 1.0.5
- CheatDeck - 1.1.2-cb3cea0
- SteamGridDB - 1.6.0
- ProtonDB Badges - 1.1.0
- Storage Cleaner - 1.4.0

## Issues
**Major blocking issue(s)**
None

**Minor non-blocking issue(s)**
None

## Closing Thoughts/Remarks
Was able to sign into GitHub, though I'd like if there were clearer wording on how to. Reports showed both in and out of game. LGTM.

## Deploy to Prod?
**Yay**

EMERALD0874 commented:
@MegalonVII I'll accept your test report since you provided all the necessary information, but please use the following template for future reports. https://wiki.deckbrew.xyz/en/user-guide/testing
